src/AppInstallerCommonCore/Certificates.cpp

```diff
@@ -546,4 +575,27 @@ namespace AppInstaller::Certificates
 
         return true;
     }
+
+    bool PinningConfiguration::LoadFromTrustedRemoteSettings()

```

Comment:
All of this JSON certificate pinning configuration load was previously test code.  This is making it product code; it probably should have tests added (for the JSON parsing part, not strictly targeting this that reads from the trusted settings).